### ISSUE TITLE
Learning rate optimization diagrams not shown in 04_mnist_basics.ipynb

### ISSUE THREAD

**Issue by tmabraham** [fastai/fastbook#89]
The graphs are missing but the caption is there:

> Now we look to see what would happen if we increased or decreased our parameter by a little bit — the adjustment. This is simply the slope at a particular point:
> 
> **A graph showing the squared function with the slope at one point**
> 
> We can change our weight by a little in the direction of the slope, calculate our loss and adjustment again, and repeat this a few times. Eventually, we will get to the lowest point on our curve:
> 
> **An illustration of gradient descent**

It looks like the image links are broken?

**Comment by tmabraham** [comment]
@sgugger A lot of the diagrams are still broken.

**Comment by sgugger** [comment]
None of the graphs are missing, I don't know what you are using to view the notebooks. You won't see them on GitHub, cause Github sucks at rendering notebooks. Use nbviewer, or pull the repo and view them in your own jupyter notebook launched.

**Comment by tmabraham** [comment]
@sgugger Oh I was not aware that GitHub was going to fail to properly render the notebooks. Sorry for the confusion and thanks for the clarification!

**Comment by sgugger** [comment]
You really want to check nbviewer then. See for instance notebook 4 on it [here](https://nbviewer.jupyter.org/github/fastai/fastbook/blob/master/04_mnist_basics.ipynb).
